cli/__init__.py

```diff
@@ -19,3 +19,9 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
+import os
+
+hashseed = os.getenv("PYTHONHASHSEED")
+if not hashseed:
+    # We replace the seed if it does not exist to make subprocesses predictable
+    os.environ["PYTHONHASHSEED"] = "42"
```

Comment:
I understand the suggestion here I think this does not quite get to the right result either since there is a `"random"`, `None`, and `int` hash seed. `"random"` is important since it allows opt-in on random hash.

```suggestion
# PYTHONHASHSEED must be "random", unset (default `None`), or an integer in range [0; 4294967295] 
_hashseed = os.getenv("PYTHONHASHSEED")
if _hashseed is None:
    _hashseed = 42
    # We replace the seed if it does not exist to make subprocesses predictable
    os.environ["PYTHONHASHSEED"] = f"{42}"
```